adapters/Vungle/VungleAdapter/GADMAdapterVungleBanner.m

```diff
@@ -50,14 +48,21 @@ - (nonnull instancetype)initWithGADMAdNetworkConnector:(nonnull id<GADMAdNetwork
   return self;
 }
 
+- (void)dealloc {
+  _adapter = nil;
+  _connector = nil;
+  _bannerAd = nil;
+  _bannerView = nil;
+}
+
 - (void)getBannerWithSize:(GADAdSize)adSize {
   id<GADMAdNetworkConnector> strongConnector = _connector;
   id<GADMAdNetworkAdapter> strongAdapter = _adapter;
   if (!strongConnector || !strongAdapter) {
     return;
   }
 
-  _bannerSize = GADMAdapterVungleAdSizeForAdSize(adSize);
+  _bannerSize = [self filterValidAdSizes:adSize];
```

Comment:
Why do we have the below for bidding but not waterfall? Can we just include `vungleAdSizeForAdSize` in utils and use it for both bidding and waterfall?
`_bannerSize = [self vungleAdSizeForAdSize:adConfiguration.adSize];`

Also the method name with `filter` sounds odd to me.

Reply:
Ok. Done.